core/server/data/schema/schema.js

```diff
@@ -173,7 +173,7 @@ module.exports = {
             type: 'string',
             maxlength: 50,
             nullable: false,
-            defaultTo: 'core',
+            defaultTo: '',
```

Comment:
Don't think there should be any default in this case at all. If it were to default to something it should at least be a value form `validations`. Removing `defaultTo: ''` parameter should be fine :+1: